org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CodeActionHandlerUseWorkspaceEditTest.java

```diff
@@ -0,0 +1,187 @@
+/*******************************************************************************
```

Comment:
You should add your tests to https://github.com/eclipse/eclipse.jdt.ls/blob/master/org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CodeActionHandlerTest.java

Reply:
I am not quite familiar with JDT test suite. How am I supposed to merge these tests into `CodeActionHandlerTest.java`, considering that these depend on `initPreferenceManager` to mock the client capabilities?

Reply:
+1 to merge it to CodeActionHandlerTest.java

Here is a sample to mock the client capabilities inside your test case.
```java
	@Test
	public void testCodeActionLiteral() throws Exception {
		when(preferenceManager.getClientPreferences().isSupportedCodeActionLiteral()).thenReturn(true);
		// Add your test case here.
	}
```

Reply:
Tests are now merged into `COdeActionHandlerTest.java`.